packages/test-scenarios/__tests__/scenario-test.js

```diff
@@ -6,12 +6,19 @@ const { merge } = require('lodash');
 jest.setTimeout(500000);
 
 async function classic(project) {
+  project.pkg.scenarioTesterTests = {
```

Comment:
I'll just make one general comment for all these scenarios, and I'm sure you're aware, but they're not complete. They need to match the specific variations we identified as cases we need to support.

Reply:
Just re-read your comment above about adding follow ons. I'd do that but write tests for the internal utils.